client/components/inline-notice/index.tsx

```diff
@@ -32,13 +42,27 @@ interface InlineNoticeProps extends Notice.Props {
 	 * @default undefined
 	 */
 	buttonVariant?: ButtonVariant;
+
+	/**
+	 * Whether to use the bundled WordPress components.
+	 *
+	 * @default false
```

Comment:
There's a mismatch with the actual value being defaulted to
```suggestion
	 * @default true
```

Reply:
d6c07d46c4f2317c7a54eb97da491a4504cfb651